_episodes/03-monitor-the-model.md

```diff
@@ -404,7 +409,8 @@ As a result, it makes much more accurate predictions on the training data than o
 Overfitting also happens in classical machine learning, but there it is usually interpreted as the model having more parameters than the training data would justify (say, a decision tree with too many branches for the number of training instances). As a consequence one would reduce the number of parameters to avoid overfitting.
 In deep learning the situation is slightly different. It can - as for classical machine learning - also be a sign of having a *too big* model, meaning a model with too many parameters (layers and/or nodes). However, in deep learning higher number of model parameters are often still considered acceptable and models often perform best (in terms of prediction accuracy) when they are at the verge of overfitting. So, in a way, training deep learning models is always a bit like playing with fire...
 
-## Set expectations: How difficult is the defined problem?
+## 9. Tune hyperparameters
```

Comment:
Like we discussed I feel it would more sense to rename this step to 'Refine the model'. 

I think in machine learning practice hyperparameter tuning or hyperparameter optimization is an automated process of finding the best hyperparameters. The simplest way to to this is a grid search. See https://en.wikipedia.org/wiki/Hyperparameter_optimization

'Refining the model' would feel better in this case, because we are **manually** changing hyperparameters of the model (with a conceptual framework in mind). Of course the automated hyperparameter tuning is a subset of 'finetuning the model'.

If you agree, you can create an issue for renaming this step everywhere in the lesson material.